integrated_channels/sap_success_factors/utils.py

```diff
@@ -322,3 +319,33 @@ def transform_language_code(code):
     language_family = SUCCESSFACTORS_OCN_LANGUAGE_CODES[language_code]
     language_name = language_family.get(country_code, language_family['_'])
     return language_name
+
+
+def get_course_metadata_for_inactivation(course_id, enterprise_customer):
+    """
+    Provide the minimal course metadata structure for updating a course to be inactive.
+    """
+    provider_id = apps.get_model(
+        'sap_success_factors',
+        'SAPSuccessFactorsGlobalConfiguration'
+    ).current().provider_id
```

Comment:
Any chance we can pass the provider_id into this function and avoid making this call for every course?